### PR TITLE
PLT-2533 Add the ability to reset a user's MFA from the system console

### PR DESCRIPTION
**Pull request by jwilander**
Assigned to CH as it's going into his branch.

### CODE REVIEW

File: webapp/client/client.jsx [hunk 1]
```diff
@@ -376,6 +376,19 @@ export default class Client {
         this.track('api', 'api_teams_signup');
     }
 
+    adminResetMfa = (userId, success, error) => {
```
Comment (coreyhulen) [inline webapp/client/client.jsx]: Can we add a new client side unit test even if it's just  err="not-enabled"

File: api/user.go [hunk 2]
```diff
@@ -1059,10 +1059,11 @@ func getProfiles(c *Context, w http.ResponseWriter, r *http.Request) {
 			if c.IsSystemAdmin() {
 				options["fullname"] = true
 				options["email"] = true
+			} else {
+				p.ClearNonProfileFields()
```
Comment (coreyhulen) [inline api/user.go]: Does the system admin need these fields?  also what about getuser() webservice call?  should we return the private mfa code stuff?

File: webapp/client/client.jsx (hunk 1 above)
Comment (jwilander) [inline webapp/client/client.jsx]: Added

**Comment by hmhealey** [comment]
+1

**Comment by coreyhulen** [comment]
+1

**Pull request merged**